index.html

```diff
@@ -439,6 +289,12 @@ <h2 class="title">Contact me</h2>
         </div>
       </section>
     </main>
+    <!-- Project popup -->
+    <section class="projectPopup" id="projectPopup"></section>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_185](https://user-images.githubusercontent.com/69638013/168415050-978fe6b1-3347-47b0-9013-5baa827e31ac.png)  | ![Screenshot_186](https://user-images.githubusercontent.com/69638013/168415149-f0e251b3-0952-4f18-9427-6b01cc98f611.png)

- [ ] Kindly adjust the size of image to cover the entire width so it looks like the Figma design
- [ ] Please adjust all the contents of the popup like the paragraph text, and spacing between each element so all content is seen on the page as your pop up is currently no scrollable currently, the buttons is not visible at 100% when the screen size is at or below 1440px
- [ ] Please add the blur background to your background of the modal this will make your portfolio look exactly like the Figma design 👍🏿